lib/dor/workflow/client/workflow_routes.rb

```diff
@@ -278,9 +278,19 @@ def process(pid:, workflow_name:, process:)
         # @param [String] repo The repository the object resides in.  The service recoginzes "dor" and "sdr" at the moment
         # @param [String] druid The id of the object to delete the workflow from
         # @param [String] workflow The name of the workflow to be deleted
+        # @param [Integer] version The version of the workflow to delete
         # @return [Boolean] always true
-        def delete_workflow(repo, druid, workflow)
-          requestor.request "#{repo}/objects/#{druid}/workflows/#{workflow}", 'delete'
+        def delete_workflow(repo, druid, workflow, version: nil)
+          qs_args = []
```

Comment:
Are we anticipating more than just version in `qs_args`? If so, this LGTM, otherwise you could probably simplify this by rolling the if/else into a guard clause and just use `version` on line 292 to address the code climate complexity warning.

Reply:
Thanks @aaron-collier I've made that change.